README.md

```diff
@@ -36,6 +36,27 @@ yarn build
 
 Strapi gives you many possible deployment options for your project including [Strapi Cloud](https://cloud.strapi.io). Browse the [deployment section of the documentation](https://docs.strapi.io/dev-docs/deployment) to find the best solution for your use case.
 
+# 🚀 Getting started with Gitpod
+Gitpod provides a fully automated development environment for your Strapi project, and the development environment is set up with just a single click. Follow these steps to get started:
+
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/distributeaid/aggregated-public-information)
+
+1. Click the ``Open in Gitpod`` button above. Note: you'll need to have an account on [Gitpod](https://gitpod.io/login/) before proceeding with the next steps (this requires a GitHub account)..
+2. Click the `Continue` button.
+3. Relax, a development environment is being set up for you in the first terminal. There's currently a bug that will stop the setup from being completed, but we'll fix that in the next steps.
+4. Create a new file called `.env` to store the environment variables.
+5. Copy and paste the content from the `.env.example` file into the `.env` file you just created.
+6. Open a new terminal and paste this code inside `openssl rand -base64 16`. This generates a secret key that will be used inside the env file.
+
```

Comment:
**Suggestion:** Since the environment variable setup applies to local development as well as gitpod, this information about populating the `.env` file may be better placed outside of the gitpod specific section of this document.

Reply:
Hi @madcalf 
Thanks for the review
I saw the issue you opened and I think you're correct. Would you prefer I make those changes in this PR?

Reply:
@Lovelyfin00 either way should work!  Let me know if I should merge this in and then you iterate on it to implement @madcalf's suggestion, or if you'd like to just address it in this PR.